.env.backups/.env.20260331_111006

```
@@ -0,0 +1,22 @@
+# Nerve Configuration
+
+# ─── Server ───────────────────────────────────────────────────────────────────
+PORT=3080
+HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)